dts/imx8mm-cgtsx8m-ultimain5.0-lvds-1024x600.dts

```diff
@@ -150,6 +150,36 @@
             linux,default-trigger = "default-on";
         };
     };
+
+    cabin_light_2: pca9633@62 {
+        compatible = "nxp,pca9633";
+        #address-cells = <1>;
+        #size-cells = <0>;
+        reg = <0x62>;
+        nxp,totem-pole;
+        nxp,inverted-out;
+        nxp,hw-blink;
+        white4 {
+            label = "white4:cabin_light";
```

Comment:
Should this label then be: `white4:cabin_light_2` or is this for compatibility purposes

Reply:
Yes, this is the "endpoint" of the node - the filename created in the sysfs. I kept the same name so no matter which device we use (PCA9632 or PCA9633), the endpoints will have the same name, so no need to handle this in the application layer.